pytorch/scripts/docker_build.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
This file shouldn't be here but that's okay for now.   We need to clean up this repo.  I'll do it after merging the cookbook branch into master.